go/test/endtoend/vreplication/cluster_test.go

```diff
@@ -933,6 +933,19 @@ func (vc *VitessCluster) startQuery(t *testing.T, query string) (func(t *testing
 	return commit, rollback
 }
 
+func killBinaries() {
+	binaries := []string{"mysqld_safe", "mysqld", "etcd", "mysqlctl", "mysqlctld", "vtgate", "vttablet", "vtctld", "vtctl", "vtorc"}
+	for _, bin := range binaries {
+		cmd := exec.Command("pkill", fmt.Sprintf(".*%s.*", bin))
+		err := cmd.Run()
+		if err != nil {
+			log.Infof("Error killing %s: %v", bin, err)
```

Comment:
Log lines are for checking if we have residual processes. If this works we can remove the log lines asap.